docs/deployment/index.rst

```diff
@@ -0,0 +1,266 @@
+.. _Deployment:
+
+FLINT.Cloud Deployment
+####################
+
+This section guides you through the process of deploying FLINT.Cloud to
+your own server, through a public cloud service provider. We are using
+Google Cloud Platform (GCP) and Azure, as of now, to deploy FLINT.Cloud
+APIs to be consumed for your FLINT-specific applications, like FLINT-UI.
+This guide assumes that you have some preliminary understanding of the
+concepts of GCP and Azure and have deployed an application before.
+
+Google Cloud Platform
+---------------------
+
+To deploy FLINT.Cloud to Google Cloud Platform, we provide a
+production-grade setup with a Layered Architecture Setup on top of the
+Google Cloud. This setup includes the usage of `Terraform`_, an
+Infrastructure-as-a-Code tool, to deploy the infrastructure and the
+application to be used.
+
+To deploy the FLINT.Cloud to Google Cloud Platform, follow these steps:
+
+1. Create a GCP service account with project owner permissions in your
+   project. This is used by Terraform to provision all the necessary
+   resources.
+2. Copy ``main.tf`` from the ``layered`` directory of this repository to
+   your Cloud Console machine.
+3. In ``main.tf``, change the project variable to your ``project ID``.
+   Change any other variables if necessary.
+4. Download the key for the service account created in the ``Step 1``
+   (in JSON format) to your project's Cloud Console machine. Rename it
+   as ``service_account.json``.
+5. Run ``terraform apply``. After this command finishes, it should
+   output the URL to FLINT Cloud (ingress).
```

Comment:
```suggestion
5. Run ``terraform apply``. 

    After this command finishes, it outputs the URL to FLINT Cloud (ingress).
```
It's clearer for the reader if you put an expected result of completing an instruction on a separate line.